### PR TITLE
feat: Initial VersionInfo component

### PR DESCRIPTION
**Pull request by AWildErin**
Requires a red pr to be merged before hand

### CODE REVIEW

File: styles/components/versioninfo.sass [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.VersionInfo 
+	flow-children: right
+	&-Text
```
Comment (craftablescience) [inline styles/components/versioninfo.sass:3]: nit: theres a `-` in the class name (😭)

File: scripts/components/versioninfo.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	static {
+		$.GetContextPanel().jsClass = this;
+	}
+
+	static onLoad() {
+		this.update();
+	}
+
+	static update() {
+		const cp = $.GetContextPanel();
```
Comment (craftablescience) [inline scripts/components/versioninfo.js:13]: as we all know, cp stands for context panel, cyberpunk, and child psychology, nothing else

File: layout/components/versioninfo.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<root>
+	<styles>
+		<include src="file://{resources}/styles/components/versioninfo.sass" />
+	</styles>
+	<scripts>
+		<include src="file://{resources}/scripts/components/versioninfo.js" />
+	</scripts>
+
+	<Panel class="VersionInfo" onload="VersionInfo.onLoad()">
```
Comment (craftablescience) [inline layout/components/versioninfo.xml:9]: can just say `update` here and get rid of `onLoad`

**Review by craftablescience — CHANGES_REQUESTED** [review]

File: layout/components/versioninfo.xml (hunk 3 above)
Comment (AWildErin) [reply layout/components/versioninfo.xml:9]: Can do. I'll sort it out when I get home.

**Comment by AWildErin** [comment]
Closing in favour of a new branch off of main. I tried rebasing this but it was complaining about commits that were not even committed from this branch.

**Pull request closed without merging**